http://bugs.gentoo.org/134690
http://gcc.gnu.org/PR27790

2006-06-07  Uros Bizjak  <uros@kss-loka.si>

	PR target/27790
	* config/i386/i386.c (ix86_expand_int_vcond): Force cop0
	into register for (code == GTU).

--- gcc/config/i386/i386.c	2006-06-04 00:50:15.000000000 -0400
+++ gcc/config/i386/i386.c	2006-06-04 00:53:32.000000000 -0400
@@ -11049,6 +11049,8 @@
      tricks to turn this into a signed comparison against 0.  */
   if (code == GTU)
     {
+      cop0 = force_reg (mode, cop0);
+
       switch (mode)
 	{
 	case V4SImode:
